x-pack/auditbeat/module/system/package/rpm_others.go

```diff
@@ -0,0 +1,13 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+// +build !linux !cgo
```

Comment:
Should be `// +build !linux,!windows !cgo,!windows`, I think. The definition for `Package` is not available for Windows because `package.go` is `// +build !windows`.

Reply:
Thanks, trying it as an extra `// + build !windows` line, I think that should work as well.